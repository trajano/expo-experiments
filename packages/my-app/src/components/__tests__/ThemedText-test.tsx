import { render } from '@testing-library/react-native';
import { ThemedText } from '../ThemedText';

it('renders correctly', () => {
  const { toJSON } = render(<ThemedText>Snapshot test!</ThemedText>);
  expect(toJSON()).toMatchSnapshot();
});
