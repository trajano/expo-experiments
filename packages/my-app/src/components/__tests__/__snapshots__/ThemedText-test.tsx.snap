// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Text
  style={
    {
      "color": "#11181D",
      "fontFamily": "ComicNeue_400Regular",
      "fontSize": 16,
      "fontStyle": undefined,
      "fontWeight": undefined,
      "lineHeight": 24,
    }
  }
>
  Snapshot test!
</Text>
`;
